frontend/src/api/__tests__/index.test.js

```
@@ -36,6 +36,13 @@ jest.mock('../misc', () => ({
   }
 }));
 
+jest.mock('../grievances', () => ({
```

ESLint

[error] 'jest' is not defined.

(no-undef)